media/stylus/base/utilities.styl

```diff
@@ -148,3 +148,15 @@ media-stack() {
     }
 }
 
+/* elements with this class move spans off screen and only display icons */
+.only-icon {
+    {$selector-icon} {
+        margin-left: 0 !important;
+        margin-right: 0 !important;
+        margin-top: -1px !important;
```

Comment:
Not sure why the !importants were needed originally, a little afraid to alter this code (copied from line 59 of forms.styl without being altered).